File: UsersModule/src/test/java/dawid/kotarba/users/UsersModuleTests.java
package dawid.kotarba.users;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.boot.test.SpringApplicationConfiguration;
import org.springframework.context.annotation.Profile;
import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;

@RunWith(SpringJUnit4ClassRunner.class)
@SpringApplicationConfiguration(classes = UsersModule.class)
@Profile("test")
public class UsersModuleTests {

    @Test
    public void contextLoads() {
    }

}

File: UsersModule/src/test/scala/dawid/kotarba/users/UsersModuleTests.scala
package dawid.kotarba.users

import dawid.kotarba.users.config.MainConfig
import org.junit.Test
import org.junit.runner.RunWith
import org.springframework.boot.test.SpringApplicationConfiguration
import org.springframework.test.context.ActiveProfiles
import org.springframework.test.context.junit4.SpringJUnit4ClassRunner

/**
  * Created by Dawid on 01.07.2016.
  */
@RunWith(classOf[SpringJUnit4ClassRunner])
@SpringApplicationConfiguration(classes = Array(classOf[MainConfig]))
@ActiveProfiles(Array("test"))
class UsersModuleTests {
  @Test def contextLoads {
  }
}
